Cabal/doc/getting-started.rst

```diff
@@ -0,0 +1,159 @@
+Getting Started with Haskell and Cabal
+======================================
+
+Installing the Haskell toolchain
+--------------------------------
+
+To install the Haskell toolchain follow the `ghcup instructions
+<https://www.haskell.org/ghcup/>`__ if you're using Linux or Mac, or follow
+`this guide <https://hub.zhox.com/posts/introducing-haskell-dev/>`__ if you're
+using Windows.
+
+
+Creating a new application
+--------------------------
+
+Let's start by creating a simple Haskell application from scratch where we'll
+learn about a Haskell package's directory structure, how to run the executable,
+and how to add external dependencies.
+
+
+Initializing the application
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start by creating a ``myfirstapp`` directory to hold the project files:
+
+.. code-block:: console
+
+    $ mkdir myfirstapp && cd myfirstapp
+
+Once you have an empty directory we can initialize our package:
+
+.. code-block:: console
+
+    $ cabal init --cabal-version=2.4 --license=NONE -p myfirstapp
+
+This will generate the following files:
+
+.. code-block:: console
+
+    $ ls
```

Comment:
`ls` would work in powershell, in dos we have to use `dir` (the output will be different in any case but whatever)

Reply:
It looks like that if we ask windows users to use *PowerShell* than there're only small tweaks needed? casing for `cmd` would make it all too messy and hard to follow.

Reply:
yeah, powershell is fine nowadays imo, it is the default console in windows 10

Reply:
Added some words about it working on unix shells and PowerShell if you're a Windows user.